test/helper/helper.go

```diff
@@ -0,0 +1,95 @@
+package helper
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	openapiclient "github.com/equinix-labs/metal-go/metal/v1"
+)
+
+func TestClient() *openapiclient.APIClient {
+	configuration := openapiclient.NewConfiguration()
+	configuration.AddDefaultHeader("X-Auth-Token", os.Getenv("METAL_AUTH_TOKEN"))
+	apiClient := openapiclient.NewAPIClient(configuration)
+	return apiClient
+}
+
+func Create_test_project(name string) string {
+	TestApiClient := TestClient()
+
+	projectCreateFromRootInput := *openapiclient.NewProjectCreateFromRootInput(name) // ProjectCreateFromRootInput | Project to create
+	include := []string{"Inner_example"}
```

Comment:
There's a lot of unnecessary copy-pasted example code here; none of the `include` or `exclude` variables in this file are set to meaningful values, so they can be removed, as can the calls to the `Include` and `Exclude` functions.